<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="gv_img">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_file_path">
      <property name="text">
       <string>Путь к текщему файлу:
</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_select_file">
      <property name="text">
       <string>Выбрать файл</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_status">
      <property name="text">
       <string>Файл не выбран</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="te_hidden_text">
      <property name="placeholderText">
       <string>Здесь отобразится спрятанный в файле текст. Его можно изменить и записать обратно в файл по клавише &quot;Записать&quot;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chb_alpha">
      <property name="text">
       <string>Enable alpha channel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_write">
      <property name="text">
       <string>Записать</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_progress">
      <property name="text">
       <string>Прогресс</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
